src/_guides/libraries/useful-libraries.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: default
+title: "Useful Dart Libraries You Should Know About"
+description: "What are some of the most useful and popular Dart libraries and where can you learn more."
+---
+
+You can leverage a wide array of available Dart libraries and packages when
+writing your code. This document shares some of the most widely
+used libraries and library packages and tells you where you can learn more.
+
+<aside class="alert alert-info" markdown="1">
```

Comment:
I'd like to deemphasize packages in the first paragraph, and then convert this note into a paragraph that discusses the two ways they might get a library (SDK and package). You _could_ also introduce pub, api.dartlang.org, and dartdocs.org.

Whatever you write, be sure it allows for exceptions to the rule.  (How does Flutter handle libraries? Also, organizations have internal libraries/packages that they don't upload to pub.dartlang.org.) You could mention/allude to those exceptions, while making clear that this doc is concerned with core libraries and libraries distributed in packages.